modules/jupyter/tests/test_hub.py

```diff
@@ -4,6 +4,12 @@
 
 from packaging.version import Version as V
 
+config_file = "../jupyter_config/config-selfauth.yaml"
```

Comment:
minor improvement for a follow-up PR, it'll be cleaner to just pass in the path to the config file as an argument

Reply:
Noted. Will take it up for next PR